app/controllers/assets_controller.rb

```diff
@@ -176,15 +176,12 @@ def update
     @asset.update_attribute(:is_spam, @asset.spam?) # makes an api call
     @asset.publish! if params[:commit] == 'Publish'
 
-    if request.xhr?
-      result ? head(:ok) : head(:bad_request)
+    if result
```

Comment:
@sudara this will be fixing #531. 
I think since we are handling all of our flash messages through `flash[:]`, it might be good to stay consistent? Could def add stimulusjs if you'd prefer to start moving messages to it.

Reply:
The forms on `mass_edit` are currently calling `update` via ajax and saving in place. We want to keep that behavior, as the point of the page is to be able to dance around your tracks and update descriptions. 

So we might want to address https://github.com/sudara/alonetone/issues/531 separately via adding a stimulus controller and a UX pattern similar to comments (spinner + confirmation that save happened). 

Alternatively, since this comment we added a stimulus controller called `flash_controller.js` for ajax flash messages, we could use that too.

Reply:
Got it